src/features/navbar/components/UserMenu.tsx

```
@@ -170,6 +171,18 @@ export function UserMenu() {
             </DropdownMenuItem>
           )}
 
+          <DropdownMenuItem asChild>
+            <Link
+              href={`https://t.me/${process.env.NEXT_PUBLIC_TELEGRAM_BOT_SLUG}?start=start`}
+              onClick={() => {
+                posthog.capture('clicked_telegram bot_user menu');
+              }}
+              className="text-sm tracking-tight text-slate-500"
+            >
+              <span>Telegram Bot<Badge className='ml-1'>New</Badge></span>
```

ESLint

[error] Replace `Telegram·Bot<Badge·className='ml-1'>New</Badge>` with `⏎················Telegram·Bot<Badge·className="ml-1">New</Badge>⏎··············`

(prettier/prettier)